src/Greenshot.Editor/Drawing/ImageContainer.cs

```diff
@@ -179,16 +179,19 @@ private void DisposeShadow()
         /// <param name="matrix"></param>
         public override void Transform(Matrix matrix)
         {
-            int rotateAngle = CalculateAngle(matrix);
-            // we currently assume only one transformation has been made.
-            if (rotateAngle != 0)
+            if (image != null)
```

Comment:
I'd prefer a fail fast, so if (image == null) return;
This is the same for the rotateAngle (my mistake)
This would prevent a lot of nesting.

Reply:
Ok but we need to call base `base.Transform(matrix);` isn't it?